osbs/constants.py

```diff
@@ -71,3 +71,5 @@
 # Logging format used in Atomic Reactor
 ATOMIC_REACTOR_LOGGING_FMT = \
     '%(asctime)s platform:%(arch)s - %(name)s - %(levelname)s - %(message)s'
+
+REPO_CONFIG_FILE = '.osbs-repo-config'
```

Comment:
(not related to this PR) Do we want to bring all the repo configs in one file eventually?